### ISSUE TITLE
[BUG] Input Chinese Cause Errors when input them into `new column`

### ISSUE THREAD

**Issue by Quorafind** [trey-wallis/obsidian-notion-like-tables#170]
**Describe the bug**

**To Reproduce**

1. Click on a new Button
2. Click on the input box
3. Input Chinese

**Expected behavior**

When composed PinYin for Chinese Character, the input box should not duplicate the English character

**Screenshots**

https://user-images.githubusercontent.com/13215013/169223361-fb702990-7d3c-4aeb-9c8f-9be61cae4d4c.mp4

**Debug Info**

```
SYSTEM INFO:
	Obsidian version: v0.14.10
	Installer version: v0.14.6
	Operating system: Windows 10 Pro for Workstations 10.0.19043
	Login status: logged in
	Catalyst license: vip
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: Minimal
	Snippets enabled: 13
	Safe mode: off
	Plugins installed: 100
	Plugins enabled: 83
		1: Workbench
		2: Hotkeys for starred files and searches
		3: Tag Wrangler
		4: Text expand
		5: Hotkeys for templates
		6: Privacy Glasses
		7: Maximise Active Pane
		8: QuickAdd
		9: MetaEdit
		10: Pandoc Plugin
		11: Hotkey Helper
		12: Hotkeys for specific files
		13: Longform
		14: Templater
		15: Calendar (Beta)
		16: Readwise Official
		17: Customizable Sidebar
		18: Folder Note Core
		19: Tag Page Preview
		20: URI Commands
		21: Excalidraw
		22: Admonition
		23: Style Settings
		24: Obsidian42 - BRAT
		25: Customizable Menu
		26: AidenLx's Folder Note
		27: Copy Block Link
		28: Kanban
		29: Icon Shortcodes
		30: Dictionary
		31: Task Changer
		32: Link Favicons
		33: Icon Folder
		34: Hider
		35: Sortable
		36: Outliner
		37: Global Hotkeys
		38: Minimal Theme Settings
		39: Better Command Palette
		40: Supercharged Links
		41: Second Window
		42: Settings Search
		43: Various Complements
		44: Dynamic Highlights
		45: Customizable Page Header and Title Bar
		46: Local REST API
		47: RSS Reader
		48: Diagrams
		49: Dialogue
		50: AidenLx's Folder Note - folderv component
		51: Query Control
		52: Zoom
		53: Search on Internet
		54: Creases
		55: Auto Link Title
		56: Footnote Shortcut
		57: Jump to link
		58: Obsidian Charts
		59: Sliding Panes (Andy's Mode)
		60: Spaced Repetition
		61: Incremental Writing
		62: Natural Language Dates
		63: Snippetor
		64: Custom Frames
		65: Version History Diff
		66: Hover Editor
		67: Banners
		68: Dataview
		69: Obsidian Memos
		70: Word Splitting for Simplified Chinese in Edit Mode
		71: Omnisearch
		72: Doubleshift
		73: Zotero Desktop Connector
		74: Pandoc Reference List
		75: Markdown Table Editor
		76: Text Format
		77: DB Folder
		78: Lapel
		79: Excel to Markdown Table
		80: Bartender
		81: Pane Relief
		82: Quick Explorer
		83: Notion-Like Tables

RECOMMENDATIONS:
	Custom theme: for cosmetic issues, please first try updating your theme to latest. If still not fixed, please try to make the issue happen in the help vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the help vault or disable community plugins.
```

**Using Mobile App?**

No

**Comment by MugenLee** [comment]
Yes, I had this problem too
@trey-wallis

**Comment by lefinite** [comment]
+1

**Comment by trey-wallis** [comment]
@Quorafind @MugenLee @lefinite  Fixed in 4.2.4 :)
